services/api/database/migrations/20220908065119_initial_db.js

```diff
@@ -126,17 +126,17 @@
                 table.integer('environment');
                 table.string('severity', 300).defaultTo('');
                 table.decimal('severity_score', 1, 1).defaultTo(0);
-                table.string('identifier', 300).notNullable();
-                table.string('lagoon_service', 300).defaultTo('');
+                table.string('identifier', 100).notNullable();
```

Comment:
i have no idea what the `identifier` field is used for to know if this will have any other impacts by reducing down to 100